ko/django_forms/README.md

```diff
@@ -389,7 +384,7 @@ https://docs.djangoproject.com/en/1.11/topics/forms/
 
 세부 페이지에 있는 수정 아이콘이 기억나죠? 이번에도 동일하게 다른 사람들이 게시글을 수정하지 못하게 할 거에요.
 
-`blog/templates/blog/post_detail.html`파일을 열어 아래 내용을 추가하세요.
+`blog/templates/blog/post_detail.html`파일을 열어 아래와 같이 작성된 라인을 찾아주세요:
```

Comment:
[Fix mistranslation]
Not adding code, just finding.
Original text : `Open blog/templates/blog/post_detail.html in the code editor and find this line:`

Reply:
thanks to your opinion :)